pkg/cmd/pipelinerun/describe.go

```diff
@@ -0,0 +1,140 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipelinerun
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"io"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+	"text/tabwriter"
+)
+
+const (
```

Comment:
Perhaps consider using a  go template, which would be easier to modify ? i.e: https://golang.org/pkg/text/template/

Reply:
agreed :)

Reply:
that said, as it's done also in `pipeline describe`, we can do a follow-up for the two :wink:

Reply:
Created issue https://github.com/tektoncd/cli/issues/46